docs/code_of_conduct.md

```diff
@@ -0,0 +1,14 @@
+# The Systemd Community Conduct Guidelines
+
+This document provides community guidelines for a safe, respectful, productive, and collaborative place for any person who is willing to contribute to systemd. It applies to all “collaborative spaces”, which is defined as community communications channels (such as mailing lists, submitted patches, commit comments, etc.).
+
+- Participants will be tolerant of opposing views.
+- Participants must ensure that their language and actions are free of personal attacks and disparaging personal remarks.
+- When interpreting the words and actions of others, participants should always assume good intentions.
+- Behaviour which can be reasonably considered harassment will not be tolerated.
```

Comment:
According to these guidelines, it seems to be totally fine to, for example, use "he" as a gender-neutral pronoun and "guys" as a general vocative in the documentation, commit messages and everywhere else. It doesn't seem to be right, given that the idea behind that issue was to, quoting the author of https://github.com/systemd/systemd/issues/10148, "send the signal that the project welcoming and inclusive".

Just to be clear, I'm not opposed to code of conducts. It's just that I think that they should be taken seriously, because they're supposed to prevent or mitigate very sensitive issues.

Reply:
I don't think it's possible to cover everything either.

> the goal of this short file is to state intent

Then probably it should be called the guideline or something like that. From my perspective, a code of conduct looks like https://wiki.postgresql.org/wiki/Code_of_Conduct (which is very thorough and transparent especially regarding the enforcement and reporting). And it also covers posting repeated comments because it's not constructive and therefore discouraged :-)

Reply:
Just to clarify, what I'm trying to say is that it's one thing to have a set of very vague rules and more or less informal way to resolve conflicts, but it's another thing to have the same set of vague rules with an enforcement clause and a team dedicated to enforce them in a way "that is deemed necessary and appropriate to the circumstances". This combo might actually scare people off and that's probably why there is no enforcement clause at https://www.ruby-lang.org/en/conduct/.

Anyway, it's possible that I'm missing something because I wasn't at the conference.